test/screenshot/mdc-fab/classes/baseline.html

```diff
@@ -1,46 +1,47 @@
 <!DOCTYPE html>
 <!--
-  ~ Copyright 2018 Google Inc. All Rights Reserved.
-  ~
-  ~ Licensed under the Apache License, Version 2.0 (the "License");
-  ~ you may not use this file except in compliance with the License.
-  ~ You may obtain a copy of the License at
-  ~
-  ~      http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
-  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  ~ See the License for the specific language governing permissions and
-  ~ limitations under the License.
-  -->
+  Copyright 2018 Google Inc. All rights reserved.
```

Comment:
Why is the capitalization here being changed when the previous capitalization matches our other files?

Reply:
The original capitalization was generated by my IntelliJ Copyright plugin.

The new lowercase version is copy-pasted from the mdc-button test pages (and is also used on the demo pages).

So all the test pages will at least be consistent.